Test/Invoke-WhiskeyNodeTask.Tests.ps1

```diff
@@ -59,9 +63,19 @@ function Invoke-SuccessfulBuild
     {
         $optionalParams['Clean'] = $True
     }
-
-    Invoke-WhiskeyNodeTask -TaskContext $WithContext -TaskParameter $taskParameter @optionalParams
-
+    if ( $npmRegistryUri )

```

Comment:
This should be a `Given` function, e.g. `GivenNpmRegistryUri'. It should set a script-level variable that this function uses.